docs/environment-variables.md

```diff
@@ -3,57 +3,57 @@
 This document outlines environment variables that can be used to customize behavior.
 
 !!! Warning
-    Environment variables are typically added to test out experimental features and should not be used by most users.
-    Environment variables may be removed at any time.
+Environment variables are typically added to test out experimental features and should not be used by most users.
+Environment variables may be removed at any time.
```

Comment:
this shouldn't be changed, it's supposed to be 4 space indented for the admonition (`!!! Warning`) [to work](https://squidfunk.github.io/mkdocs-material/reference/admonitions/#usage)

Reply:
I's sorry, this may be caused by `make docs`, I have removed this.

Reply:
It's not caused by `make docs`: the command only does linting and codegen, it does not do formatting. I suspect your editor may have tried to auto-format without understanding the `mkdocs-material` syntax.